src/constants/error.js

```diff
@@ -0,0 +1,47 @@
+const ERROR_MESSAGE_GENERATOR = Object.freeze({
+  overMaxLength(target, max) {
+    return `${target}에 ${max}자 이하의 문자열을 입력해주세요!`;
+  },
```

Comment:
저는 `format` 형식으로 사용하기보단 단순 템플릿 리터럴을 사용하여 나타내는게 조금 더 깔끔하지 않나 싶은데, 민규님 생각은 어떠신가요?

```suggestion
  overMaxLength : `${target}에 ${max}자 이하의 문자열을 입력해주세요!`
```